spec/rubocop/cop/style/if_unless_modifier_spec.rb

```diff
@@ -310,4 +310,17 @@
       expect(corrected).to eq 'puts "string", (1 if a)'
     end
   end
+
+  context 'if-end with conditional as body' do
+    let(:source) do
+      ['if condition',
```

Comment:
Happy to add it, but isn't that already covered by [this more general case](https://github.com/amuino/rubocop/blob/if-unless-modifier-with-nested-conditional/spec/rubocop/cop/style/if_unless_modifier_spec.rb#L43-L54)?